Makefile

```diff
@@ -14,60 +14,34 @@
 
 EXTENSION_PREFIX            := gardener-extension
 NAME                        := runtime-gvisor
-REGISTRY                    := eu.gcr.io/gardener-project
+NAME_INSTALLATION           := runtime-gvisor-installation
+CMD_DIRECTORY				:= ./cmd/$(EXTENSION_PREFIX)-$(NAME)
+REGISTRY                    := eu.gcr.io/gardener-project/gardener
 IMAGE_PREFIX                := $(REGISTRY)/extensions
 REPO_ROOT                   := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 HACK_DIR                    := $(REPO_ROOT)/hack
 VERSION                     := $(shell cat "$(REPO_ROOT)/VERSION")
-LD_FLAGS                    := "-w -X github.com/gardener/$(EXTENSION_PREFIX)-$(NAME)/pkg/version.Version=$(IMAGE_TAG)"
-VERIFY                      := true
-LEADER_ELECTION             := false
+LD_FLAGS                    := $(shell ./vendor/github.com/gardener/gardener/hack/get-build-ld-flags.sh github.com/gardener/gardener-extension-runtime-gvisor)
```

Comment:
Reuse $REPO_ROOT and $ETENSION_PREFIX and $NAME variables here?

Reply:
$REPO_ROOT would be go/src/user/github.com/gardener/gardener/ whereas I only want github.com/gardener?
I used $EXTENSION_PREFIX and $NAME variables

Reply:
```suggestion
LD_FLAGS                    := $(shell $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/get-build-ld-flags.sh "github.com/gardener/$(EXTENSION_PREFIX)-$(NAME)")
```